macos/Sources/Ghostty/SurfaceView_AppKit.swift

```diff
@@ -617,12 +617,19 @@ extension Ghostty {
             let mods = Ghostty.ghosttyMods(event.modifierFlags)
             ghostty_surface_mouse_pos(surface, pos.x, frame.height - pos.y, mods)
 
+            
+            let focusFollowsMouse: Bool = {
+                if let window = self.window as? TerminalWindow {
+                    return window.focusFollowsMouse;
+                } else if let window = self.window as? QuickTerminalWindow {
+                    return window.focusFollowsMouse;
+                } else {
+                    return false;
+                }
+            }()
+            
             // If focus follows mouse is enabled then move focus to this surface.
-            if let window = self.window as? TerminalWindow,
-               window.isKeyWindow &&
-                window.focusFollowsMouse &&
-                !self.focused
-            {
+            if (self.window!.isKeyWindow && !self.focused && focusFollowsMouse) {
```

Comment:
I'd prefer to guard against window being null here. It is possible (we detach views at certain times).